netdeploy/networkTemplate.go

```diff
@@ -161,6 +162,11 @@ func loadTemplate(templateFile string) (NetworkTemplate, error) {
 	}
 	defer f.Close()
 
+	if runtime.GOARCH == "arm" || runtime.GOARCH == "arm64" {
+		// for arm machines, use smaller key dilution
+		template.Genesis.PartKeyDilution = 100
```

Comment:
Does this actually help anything?  Unless the partkey is valid for fewer rounds, a `PartKeyDilution` value that's further away from sqrt(number-of-rounds) will cause more private keys to be generated, and the partkey will be larger.

Reply:
It's the bucket size that is the issue here. The cryptographic creation of all the second level keys ( i.e. RNG, SK/PK and signing ) is taking too long. ( since we're creating 10K of them )
Limiting this to 100 seems to work around the issue. ( each bucket contains keyDilution entries )

Reply:
It took me a while to figure that we will need *both* to move the disk persistence as well as reduce the key dilution before the deadlock would get eliminated. This seems to be the case where we had two triggers for the (almost) same issue.